README.md

```diff
@@ -272,7 +272,7 @@ function greet() {
 module.exports = { greet }
 ```
 
-**IMPORTANT:** Currently, a limitation in the js-pdk is that host functions must have 1 `I64` pointer param and 1 `I64` pointer result. We enforce this through validation of the typescript interface file.
+**IMPORTANT:** Currently, a limitation in the js-pdk is that host functions may only have up to 5 arguments.
```

Comment:
w/r/t both these docs changes, is there still the limitation that each function must return only a single `I64` ?

Reply:
No, they can return any native wasm type or have no return value.

Reply:
ok! i made a suggestion to just explicitly call this out. totally could be common knowledge, but since we specify the args count, i feel like it can't hurt to also specify the returns. not stuck on this though if you think its superfluous.

Reply:
I am on the fence about that since JS/TS only allow one return value anyway and mentioning it there might make people think a return value is required.